wp-offline-shell/readme.txt

```diff
@@ -43,6 +43,12 @@ It's best to update the plugin file options when you've added a new file to your
 
 Of course! Service Workers are an emerging technology so this plugin should be updated often based on feedback from developers using WordPress in different ways
 
+== Screenshots ==
+
+1. File selection screen -- the admin may choose which files to cache
```

Comment:
Are you adding the screenshots only to the SVN repo? It would be useful to also add them to this repo.